### PR TITLE
Qlik

### PR DESCRIPTION
**Pull request by talos**
@rafatower This PR is for tomorrow's discussion.  We may shift some functions around still.

The two functions that will be public-facing will be `OBS_GetDemographicSnapshot`, `OBS_GetSegmentSnapshot`.

### CODE REVIEW

File: src/pg/test/expected/40_observatory_utility_test.out [hunk 1]
```diff
@@ -0,0 +1,79 @@
+SELECT set_config(
```
Comment (rafatower) [inline src/pg/test/expected/40_observatory_utility_test.out]: @ohasselblad I see you added some fixtures and expectations to the test dir but no test file.

Comment (rafatower) [inline src/pg/test/expected/40_observatory_utility_test.out]: hold on, I think I can figure out and tackle #5

Comment (ohasselblad) [inline src/pg/test/expected/40_observatory_utility_test.out]: Oops, it looks like I didn't commit the test file. Will push once at a computer.

Comment (ohasselblad) [inline src/pg/test/expected/40_observatory_utility_test.out]: Ok, it's up there now! 
https://github.com/CartoDB/observatory-extension/commit/b3b4c8e4c303d9beea50f9c51cb0228260e38a65

**Comment by ohasselblad** [comment]
@stuartlynn, was there a plan to move the internal functions to be like `_OBS`?

**Pull request closed without merging**